### DISCUSSION TITLE
How to get the value of barcode marker In ar.js?

### DISCUSSION THREAD

**Discussion by akashkumarmandal** [AR-js-org/AR.js#679] [General]
I want to automatically popup the value of random barcode marker used in ar.js.
Kindly guide.

**Comment by akashkumarmandal** [comment]
How aframe compare the barcode value after detecting the barcode marker..?

**Comment by nickw1** [comment]
Hi @akashkumarmandal I'm unable to answer this but others maybe able to.


Many thanks!
